### DISCUSSION TITLE
Missing easyrtc.js, naming incosistencies

### DISCUSSION THREAD

**Discussion by jimver04** [networked-aframe/networked-aframe#411] [Q&A] [answered]
Hi,

In "examples/basic-audio.html",  the file with path :   <script src="/easyrtc/easyrtc.js"></script>    can not be found.

I guess that you refer to  "/server/easyrtc-server.js"
When I insert the "server/easyrtc-server.js" then I get the mysterious:

Uncaught TypeError: this.easyrtc is undefined

from two lines of  networked-aframe.js

networked-aframe.js line 217 > eval:65:7

Uncaught TypeError: this.easyrtc is undefined
    setServerUrl EasyRtcAdapter.js:65
    connect NetworkConnection.js:46
    connect networked-scene.js:60
    value Aframe
    init networked-scene.js:43
    Aframe 64

networked-aframe.js line 217 > eval:198:27

Uncaught TypeError: this.easyrtc is undefined
    broadcastData EasyRtcAdapter.js:198
    broadcastData NetworkConnection.js:180
    tick networked.js:88
    Aframe 50

​

**Comment by vincentfretin** [comment, marked as answer]
Hi, the example works as intended. 
`server/easyrtc-server.js` is the server part, you shouldn't include it in a script tag.

The file is used when you run 
`npm run dev` or `npm start`

The `easyrtc-server.js` file is a node app with the express framework that define several routes including `/easyrtc/easyrtc.js` defined by the open-easyrtc library.
The `/easyrtc/easyrtc.js` route is defined [here](https://github.com/open-easyrtc/open-easyrtc/blob/32c15ee2ea4a2f2af6faeea408a49103cd8ed877/lib/easyrtc_default_event_listeners.js#L1705). The `apiPublicFolder` that is `/easyrtc` is defined [here](https://github.com/open-easyrtc/open-easyrtc/blob/b33ee325d5ff952d177d48a2ddf8fe821aab781d/lib/easyrtc_default_options.js#L44)

You need to access the example via the server, so http://localhost:8080/basic-audio.html
The `/easyrtc/easyrtc.js` script will download the open-easyrtc lib through the node server dynamically.
